python/tvm/script/builder/tir/op.py

```diff
@@ -133,6 +133,10 @@ def handle():
     return _ffi_api.Handle()
 
 
+def if_then_else(cond, t, f, dtype):
```

Comment:
Let's also make dtype optional:

```suggestion
def if_then_else(cond, t, f, dtype=None):
```